test/mocha.opts

```diff
@@ -0,0 +1,2 @@
+--recursive
+--reporter dot
```

Comment:
`loopback-next` repo uses the default reporter. Not sure if we want to set to `dot`.

Reply:
`extension-starter` uses `dot` reporter. It keeps the output minimal. I talked to @bajtos and he suggested using the `dot` reporter and we should perhaps consider using it in `loopback-next` as well instead of the default.